src/test/java/lesson17/OrderDAOTest.java

```diff
@@ -0,0 +1,33 @@
+package lesson17;
+
+import lesson17.models.Orders;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.util.Set;
+
+import static org.junit.Assert.assertNotNull;
+
+public class OrderDAOTest {
+    private OrderDAOImpl dao = new OrderDAOImpl();
+    private static final BigDecimal id = new BigDecimal(113042);
+
+    @Test
+    public void testGetAllOrders() throws SQLException {
+        Set<Orders> actual = dao.getAllOrders();
+        assertNotNull(actual);
```

Comment:
Your tests don't cheak any. Lock at you code. If you don't have result you return new HashSet. It empty, but not null